lms/djangoapps/support/static/support/js/spec/views/certificates_spec.js

```diff
@@ -129,7 +131,7 @@ define([
             }).render();
 
             // Simulate a response from the server
-            AjaxHelpers.expectJsonRequest(requests, 'GET', '/certificates/search?query=student@example.com');
+            AjaxHelpers.expectJsonRequest(requests, 'GET', '/certificates/search?user_query=student@example.com');
             AjaxHelpers.respondWithJson(requests, SEARCH_RESULTS);
 
             // Check the search results
```

Comment:
I think that you have not yet implemented the javascript tests to cover course_id, is that correct?